recog.gemspec

```diff
@@ -31,6 +31,14 @@ Gem::Specification.new do |s|
   s.executables   = s.files.grep(%r{^recog/bin/}).map { |f| File.basename(f) }
   s.require_paths = ['lib']
 
+  gem_public_cert = ENV['GEM_PUBLIC_CERT']
+  gem_private_key = ENV['GEM_PRIVATE_KEY']
+
+  if !gem_public_cert.nil? && !gem_private_key.nil?
```

Comment:
not nil is sorta a double negative.  I think you can just do 

```
if gem_public_cert && gem_private_key
```

Reply:
@pdeardorff-r7's guidance is in alignment with 
[RuboCop: No non-nil Checks](https://github.com/rubocop/ruby-style-guide?tab=readme-ov-file#no-non-nil-checks).

Reply:
Ah, I tested both usages locally, and saw online it was mentioned that there were technically some differences between them. so thought adding the nil check would be safer. Thanks for pointing that out.